packages/manager/apps/pci-workflow/src/api/hooks/workflows.tsx

```diff
@@ -24,14 +28,18 @@ export type TWorkflow = TRemoteWorkflow & {
   lastExecutionStatus: TExecutionState;
 };
 
+const getWorkflowQueryOptions = (projectId: string, regionName: string) =>
+  queryOptions({
+    queryKey: [projectId, 'regions', regionName, 'workflows'],
+    queryFn: async () => getRegionsWorkflows(projectId, regionName),
+  });
```

Comment:
This allows to fix tanstack query keys dependencies and typing when cache data is set